Classes/WPEditorView.m

```diff
@@ -275,8 +275,17 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
             [newValue getValue:&newSize];
```

Comment:
Not really related to this PR but thought I'd mention since as I noticed it.  We could probably use `CGSize newSize  = newValue.sizeValue;` and avoid the `out` argument here.